packages/create-atomic/package.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "@coveo/create-atomic",
+  "private": true,
+  "description": "Coveo Atomic Generator",
+  "version": "1.19.0",
+  "bin": {
+    "create-atomic": "./index.mjs"
+  },
+  "main": "index.mjs",
+  "scripts": {
+    "start": "plop",
+    "npm:bump:template": "npm --no-git-tag-version version",
+    "npm:publish:template": "npm publish --access public --registry http://localhost:4873",
```

To fix the E2E for now.

```suggestion
    "notyet:npm:bump:template": "npm --no-git-tag-version version",
    "notyet:npm:publish:template": "npm publish --access public --registry http://localhost:4873",
```